internal/services/engines/java/sample_test.go

```diff
@@ -891,5 +891,299 @@ public class Foo {
 		Connection conn = DriverManager.getConnection("jdbc:derby:memory:myDB;create=true", "login", password);
 	}
 }
+`
+
+	SampleVulnerableHSJAVA150 = `
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>Log4j2Example</groupId>
+    <artifactId>Log4j2Example</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <dependencies>
+        <!--  https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-api  -->
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-core</artifactId>
+            <version>2.8.2</version>
+        </dependency>
+    </dependencies>
+</project>
+`
+
+	SampleSafeHSJAVA150 = `
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>Log4j2Example</groupId>
+    <artifactId>Log4j2Example</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <dependencies>
+        <!--  https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-api  -->
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-core</artifactId>
+            <version>2.15.0</version>
+        </dependency>
+    </dependencies>
+</project>
+`
+
+	Sample2VulnerableHSJAVA150 = `
```

Comment:
What do you think about adding `Gradle` and `Maven` suffix on these samples? Just to make more clear the different between them.

Reply:
Done